### PR TITLE
Create update setting form

### PR DESCRIPTION
**Pull request by manar-abed**
relates #57

### CODE REVIEW

File: client/src/components/Setting/Setting.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const { Item } = Form;
+  const [imgLink, setImgLink] = useState('');
+  const [cvLink, setCVLink] = useState('');
+  const [bio, setBio] = useState('');
+  const [experenice, setExperenice] = useState('');
+
+  const updateSetting = async () => {
+    const source = axios.CancelToken.source();
+    try {
+      const res = await axios.post('/api/user', {
```
Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:18]: Destructure the message

Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:18]: Destructure the message

File: client/src/components/Setting/Setting.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          <Input placeholder="Please input your img Url" value={imgLink} />
+        </Item>
+        <Item name="age" label="CV Link" type="url" value={cvLink} onChange={(e) => setCVLink(e.target.value)}>
+          <Input placeholder="Please input your CV Url" value={cvLink} />
+        </Item>
+        <Item name="bio" label="Bio" type="text" value={bio}>
+          <Input.TextArea placeholder="Please input your Bio" onChange={(e) => setBio(e.target.value)} value={bio} />
+        </Item>
+        <Item name="experenice" label="Level of Experenice">
+          <Radio.Group value={experenice} onChange={(e) => setExperenice(e.target.value)}>
```
Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:48]: Destructure the TextArea and group

Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:48]: Destructure the TextArea and group

File: client/src/components/Setting/Setting.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Form, Input, Button, message, Radio,
+} from 'antd';
+import './setting.css';
```
Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:6]: Name your files index.jsx and style.css

Comment (Raghad-Mezied) [inline client/src/components/Setting/Setting.jsx:6]: Name your files index.jsx and style.css

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Mahmoud-Ahmad2 — APPROVED** [review]

**Review by abedshamia — APPROVED** [review]

**Pull request merged**